CHANGES.rst

```diff
@@ -35,6 +35,9 @@ Changelog (Pillow)
 - Docs: Changed documentation references to 2.x to 2.7 #2921
   [radarhere]
 
+- Add a loader for the BLP format from Warcraft and World of Warcraft #3007
+  [jleclanche]
+
```

Comment:
PRs do not usually have their CHANGES entry as part of them - this is inviting merge conflicts for your PR if another PR is merged first. Whoever merges it in will add the entry.